KEYCLOAK-8139 Update supported/deprecated wildfly adapters on the dow…

mposolda opened the pull request:
…nloads page

mposolda commented:
@stianst @hmlnarik @vramik Added wildfly13 as the new adapter and deprecate 11,10,9 wildfly. See how the downloads page looks like: http://imgbox.com/EeDX4lFi

hmlnarik commented on src/main/resources/templates/downloads-11.ftl:
```diff
@@ -0,0 +1,357 @@
+                        <table class="table-downloads-inner">
+                            <tr>
+                                <td>
+                                   13<br />
```
Could the 11, 10, 9 be in separate cell (similarly to 8) instead of <br>?

mposolda replied:
@hmlnarik Maybe? Do we want to emphasize it's same ZIP for all 9-13 or not emphasize and use separate cells? I don't have strong preference. @stianst WDYT?

vramik replied:
Do we want to somehow mention we doesn't support wf12 at all?

stianst replied:
I'm happy with just <br> and showing that it's the same DL.

I'd just leave WF12 out of it. Trying to mention that would mess things up.

hmlnarik replied:
I don't have strong preference here too.

vramik replied:
I'm ok with it as it's now.

stianst commented:
You also need to bump the version in https://github.com/keycloak/keycloak-web/blob/master/src/main/resources/version-template.json. Otherwise the next release will still use the old template.

mposolda commented:
@stianst I've updated the version-template.json in this PR. Updated "downloadTemplate" from 10 to 11. I believe that's the only updated version needed?
